apps/test-app-runtime/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/node": "^18",
     "aws-embedded-metrics": "^4.1.0",
     "jest": "^29",
-    "ts-jest": "^29",
+    "ts-jest": "^29.1.0",
```

Comment:
Revert?

Reply:
Mentioned in the description, ts-jest doesn't use semantic versioning, we need the lastest version to get ts5 support, plus this is a dev dep.